keras_retinanet/bin/evaluate.py

```diff
@@ -132,23 +129,25 @@ def main(args=None):
     # print(model.summary())
 
     # start evaluation
-    if args.dataset_type == 'coco':
-        from ..utils.coco_eval import evaluate_coco
-        evaluate_coco(generator, model, args.score_threshold)
-    else:
-        average_precisions = evaluate(
-            generator,
-            model,
-            iou_threshold=args.iou_threshold,
-            score_threshold=args.score_threshold,
-            max_detections=args.max_detections,
-            save_path=args.save_path
-        )
-
-        # print evaluation
-        for label, average_precision in average_precisions.items():
-            print(generator.label_to_name(label), '{:.4f}'.format(average_precision))
-        print('mAP: {:.4f}'.format(sum(average_precisions.values()) / len(average_precisions)))
+    average_precisions = evaluate(
```

Comment:
The way it is implemented now, `evaluate_coco` isn't used. COCO has quite a nice evaluation output, but it is unfortunately incompatible with other types of datasets. Still though, for COCO I think we should use `evaluate_coco`.